local-cli/util/yarn.js

```diff
@@ -8,6 +8,7 @@
  */
 'use strict';
 
+const execSync = require('child_process').execSync;
```

Comment:
For more detail,

- As I understand, `init/init` uses `yarn.getYarnVersionIfAvailable()` to get version and it will return `null` from `catch` block due to `execSync is not defined`

```bash
This will walk you through creating a new React Native project in /Users/n3tr/Code/RN/AwesomeApp
Using yarn v0.16.1
Installing react-native... # via yarn
Setting up new React Native app in /Users/n3tr/Code/RN/AwesomeApp
Installing React... # via npm: from npm install output (init/init.js:80)
Installing Jest...
```